src/content/contribute/writing-a-scaffold.md

```diff
@@ -8,9 +8,20 @@ contributors:
   - rishabh3112
 ---
 
-Welcome to the demonstration of the new `webpack init` command! To view what we are building today, run `webpack init webpack-scaffold-demo`. This demo will show you how to build your own webpack scaffold. Let's start by creating a file named `generator.js`.
+A scaffold is a node module which exports a `yeoman generator` based upon [Scaffolding API](/guides/scaffolding/#api). Scaffolds are used to initialise a new webpack project, tailored to a specific use case. To view what we are building today, run `webpack init webpack-scaffold-demo`. This demo will show you how to build your own webpack scaffold. Let's start by creating a file named `generator.js`.
 
-## Create a basic structure
+## Quick Start
+
+[`webpack-scaffold-starter`](https://github.com/rishabh3112/webpack-scaffold-starter) can be used to setup a new scaffold project. To do so, follow the following commands.
```

Comment:
To recommend a personal project on official docs website, it'd need to get a full code review. I will not approve this nor do i have the time. maybe @evenstensberg has the time to do it and take the responsibility, especially considering this will get file system access.

Reply:
We can have it under webpack or webpack-contrib if that helps in resolving this issue.

Reply:
That would solve the concern of linking a tool from docs

Reply:
So what you want me to do in this regard?
P.S.
No offence, but as every node package can have the file system access, No personal project should be on the official docs ( even the plugins/loaders under personal accounts ) if going by the same argument.

Reply:
Yeah i dont mean any offence either, but since you are creating this for webpack-cli, why cannot it be under webpack-cli ownership, then there are no risks in such linking

Reply:
@sokra what do you think? If you say we should do it as is, i don't mind.

Reply:
>  why cannot it be under webpack-cli ownership, then there are no risks in such linking

This is package is supposed to be standalone, thus the solutions includes
1. Have under either `webpack` or `webpack-contrib`
2. Let it be as it is now

P.S.
Let me know if this PR should be closed. Thanks.